package.json

```diff
@@ -3,7 +3,15 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^3.0.2",
+    "@material-ui/icons": "^3.0.1",
+    "cldr-data": "^33.0.0",
+    "iana-tz-data": "^2017.1.0",
+    "material-ui-icons": "^1.0.0-beta.36",
```

Comment:
It looks like we have two libraries for material UI icons. Do we need both or should we remove one?